addon/components/ember-node/embedded-editor.ts

```diff
@@ -203,11 +204,21 @@ export default class EmbeddedEditor extends Component<Args> {
   @action
   onSelected() {
     if (this.args.selected && this.innerView) {
+      const lastKeyPressedPluginState = lastKeyPressedPluginKey.getState(
+        this.outerView.state
+      );
```

Comment:
I think it's better to use `this.controller.mainEditorState` here. (To ensure this will also work with multi-level nested editors in the future).

Reply:
Yes